packages/react/context-menu/src/ContextMenu.tsx

```diff
@@ -115,15 +141,14 @@ const ContextMenuTrigger = React.forwardRef<ContextMenuTriggerElement, ContextMe
       () => window.clearTimeout(longPressTimerRef.current),
       []
     );
-    const handleOpen = (event: React.MouseEvent | React.PointerEvent) => {
+    const handleOpenPointUpdate = (event: React.MouseEvent | React.PointerEvent) => {
```

Comment:
What prompted the rename/split here?

Reply:
TypeScript pleasing due to change in context methods, moot after https://github.com/radix-ui/primitives/pull/1396/commits/5570580ee93e8168d641c8d757cb650f460a437e though